### PR TITLE
Update v2 strings

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
@flodolo We have a number of string edits and a few new strings. We unanimously agree that we owe you champagne at Whistler.

Question:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; This string: 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`see-if-youve-been-part = See if you’ve been part of an online data breach.`

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Is very close to these strings:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`see-if-breached = See if you’ve been in an online data breach.`
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`scan-label = See if you’ve been involved in a data breach.`

Could we update one of the old strings and avoid creating a new string ID for this?

### CODE REVIEW

**Comment by pdehaan** [comment]
Not sure if we want to address any of these l10n issues at the same time:

- #875 &mdash; A few missing data classes in locales/en/data-classes.ftl
- #915 &mdash; “Email addresses” string reused in different contexts
- #919 &mdash; Need text for when a breach list search has no results
- #921 &mdash; Missing l10n string for "additional-information-including"

**Comment by flodolo** [comment]
> Could we update one of the old strings and avoid creating a new string ID for this?

Either we use the existing string, or we create a new ID if you want the updated text. Your choice.

We should also fix the other l10n issues that Peter pointed out, and we should do it as soon as possible (like, yesterday). In particular the one pointed out by Théo, given French is a target locale.

Any update on pushing back the launch date one week? We really need that extra time.

**Comment by lesleyjanenorton** [comment]
@flodolo It's attributable to a variety factors: 1.) Me missing things 2.) Discrepancies between the visual spec and the copy deck and 3.) The content strategist feeling that there were tweaks needed post "final copy" handover. I'll fix #915 and #875 and get those pushed up.

File: locales/en/data-classes.ftl [hunk 1]
```diff
@@ -12,6 +12,7 @@ avatars = Avatars
 bank-account-numbers = Bank account numbers
 beauty-ratings = Beauty ratings
 biometric-data = Biometric data
+bios = Bios
```
Comment (flodolo) [inline locales/en/data-classes.ftl]: Is this "bios" and in "biographies", or something else? Can we get a comment?

File: locales/en/data-classes.ftl [hunk 2]
```diff
@@ -101,6 +103,7 @@ smoking-habits = Smoking habits
 sms-messages = SMS messages
 social-connections = Social connections
 social-media-profiles = Social media profiles
+social-security-numbers = Social security numbers
```
Comment (pdehaan) [inline locales/en/data-classes.ftl]: SSNs sounds scary. Should we add it to the priority data classes list? https://github.com/mozilla/blurts-server/blob/d99aa8133628202a15ce2471869bf5e3714ed04f/template-helpers/breach-detail.js#L44

Comment (lesleyjanenorton) [reply locales/en/data-classes.ftl]: Definitely yes.

Comment (pdehaan) [reply locales/en/data-classes.ftl]: We can also do it in a separate pr, I just wanted to call it out while I saw it.
But if this PR is ready and has @flodolo’s blessing, I’d rather land it quickly and not block on the priority data class stuff.

Comment (flodolo) [reply locales/en/data-classes.ftl]: I'd really like this to land soon. But I also need a comment on the *bios* thing

Comment (pdehaan) [reply locales/en/data-classes.ftl]: @flodolo already done, see https://github.com/mozilla/blurts-server/pull/923/files#diff-bd84432b6e22fdc22c58a93217eeefa9

Comment (flodolo) [reply locales/en/data-classes.ftl]: Then yes, it should land ASAP

**Pull request merged**